src/gui/qgsdatumtransformdialog.cpp

```diff
@@ -59,32 +64,30 @@ QgsDatumTransformDialog::QgsDatumTransformDialog( const QgsCoordinateReferenceSy
   mLabelSrcDescription->clear();
   mLabelDstDescription->clear();
 
-  for ( int i = 0; i < 2; i++ )
-  {
-    mDatumTransformTreeWidget->setColumnWidth( i, settings.value( QStringLiteral( "Windows/DatumTransformDialog/columnWidths/%1" ).arg( i ), mDatumTransformTreeWidget->columnWidth( i ) ).toInt() );
-  }
-
   load( selectedDatumTransforms );
 }
 
 void QgsDatumTransformDialog::load( const QPair<int, int> &selectedDatumTransforms )
 {
-  mDatumTransformTreeWidget->clear();
+  mDatumTransformTableWidget->setRowCount( 0 );
+
+  int row = 0;
 
   for ( const QgsCoordinateTransform::TransformPair &transform : qgis::as_const( mDatumTransforms ) )
   {
-    QTreeWidgetItem *item = new QTreeWidgetItem();
     bool itemDisabled = false;
     bool itemHidden = false;
 
+    if ( transform.sourceTransformId == -1 || transform.destinationTransformId == -1 )
```

Comment:
@nyalldawson the code was like that: do not show the line if one of the transform is invalid. Does it make sense? Isn't there case where there would be only one transform? For instance if the dest or source is already on WGS84???

Reply:
Yes I think that condition is wrong

Reply:
ok fixed